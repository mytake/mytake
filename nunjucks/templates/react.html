{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
