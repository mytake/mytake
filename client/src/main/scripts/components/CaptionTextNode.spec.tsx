import * as React from "react";
import * as renderer from "react-test-renderer";
import CaptionTextNode from "./CaptionTextNode";

const captionNode = "Good evening I'm Martha Raddatz from ABC News.";

const speaker = "Raddatz";

test("CaptionTextNode", () => {
  const tree = renderer
    .create(<CaptionTextNode documentNode={captionNode} speaker={speaker} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
