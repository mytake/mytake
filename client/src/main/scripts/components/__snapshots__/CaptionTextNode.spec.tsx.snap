// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaptionTextNode 1`] = `
<div
  className="document__node"
>
  <h3
    className="document__node-text document__node-text--speaker"
  >
    Raddatz
  </h3>
  <div
    className="document__node-text-container"
  >
    <p
      className="document__node-text document__node-text--p"
    >
      Good evening I'm Martha Raddatz from ABC News.
    </p>
    <div
      className="document__node-height-div"
    >
      <p
        className="document__node-text document__node-text--hidden-p"
      />
    </div>
  </div>
</div>
`;
